scribble/app/views/posts/edit.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Edit Post</h1>
+<form action='/posts/<%= @post.id %>' method='post'>
```

Comment:
Could you use `form_for` here? For example:

https://github.com/ga-dc/tunr_rails_deployment/blob/master/app/views/artists/_form.html.erb